systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -721,6 +723,9 @@ void testRemoveUserOperatorFromEntityOperator() {
 
     @Test
     void testRemoveUserAndTopicOperatorsFromEntityOperator() {
+        // TODO temporary fix
```

Comment:
How much temporary is this? Is it still needed?

Reply:
I was going to fix this sin a followup, this PR is getting so stupidly big now is all.

Reply:
I think that is fine with me ... although I'm not sure what your plan is ... I think the assume works fine there. In any case maybe you can change these comments to make them more self-explanatory. Right now it is not clear if it is something forgotten or something planned for further PRs etc.